types/weixin-app/index.d.ts

```diff
@@ -2703,7 +2703,10 @@ declare namespace wx {
    * 调用后会立刻弹窗询问用户是否同意授权小程序使用某项功能或获取用户的某些数据，
    * 但不会实际调用对应接口。如果用户之前已经同意授权，则不会出现弹窗，直接返回成功。
    */
-  function authorize(options: AuthSetting): void;
+  interface AuthorizeOptions extends BaseOptions {
+    scope: string;
```

Comment:
seems this will be better then `string`:

```
scope: 'scope.userInfo' | 'scope.userLocation' | 'scope.address' | 'scope.invoiceTitle' | 'scope.werun' | 'scope.record' | 'scope.writePhotosAlbum' | 'scope.camera';
```

[doc](https://developers.weixin.qq.com/miniprogram/dev/framework/open-ability/authorize.html)

Reply:
and pls add more scope here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2b9d46ec7ef931d4272b99b71b975ac3eb3831e7/types/weixin-app/index.d.ts#L2897